core/src/datayoga_core/expression.py

```diff
@@ -136,17 +124,25 @@ def exec_sql(self, data: List[Dict[str, Any]], expressions: Dict[str, str]) -> L
         """
         # use a CTE to create the in memory data structure
         data_inner = flatten_data(data)
-        cte_clause = self._get_cte(data_inner)
-
+        # builds a CTE expression for fetching in memory data
```

Comment:
CTE contains expression word in it already, you can just use the full term (common table expression) or just leave CTE

Reply:
actually it is no longer a CTE